src/api/operators.ts

```diff
@@ -157,6 +157,25 @@ export const rectangle = (
     asPDFNumber(height),
   ]);
 
+/**
+ * @param x1 x coordinate of the first point
+ * @param y1 y coordinate of the first point
+ * @param x2 x coordinate of the second point
+ * @param y2 y coordinate of the second point
+ */
+export const line = (
```

Comment:
This new operator helper does not appear to be used (and I am unable to find the operator it wraps anywhere in the PDF spec). Do we need to keep this change, or can it be removed?

Reply:
OK, I will remove it.